README.Rmd

```diff
@@ -25,10 +25,12 @@ knitr::opts_chunk$set(
 
 - `get_resource()` extracts a single resource from an open dataset by resource id
 - `get_dataset()` extracts multiple resources from an open dataset by dataset name
+- `list_datasets()` returns the names of all available datasets 
+- `list_resources()` returns information on all resources within an open dataset by dataset name
 
 For extracting metadata and search functionality, we recommend using the [ckanr package](https://docs.ropensci.org/ckanr/). 
 
-`phsopendata` can be used on both the [server](https://rstudio.nhsnss.scot.nhs.uk/) and desktop versions of RStudio. However, depending on firewall settings, proxy use may need to be configured with `use_proxy()`.
+`phsopendata` can be used on both Posit Workbench and desktop versions of RStudio. However, depending on firewall settings, proxy use may need to be configured with `use_proxy()`.
```

Comment:
Maybe simpler to say it should work for PHS systems and as long as people can access the opendata platform. If another org needs to use proxies to access data I'm not sure we could help with that.